CITATION.txt

```diff
@@ -6,5 +6,5 @@ below:
   organization = {toblerity.org},
```

Comment:
What's a better organization name?

Reply:
We can maybe also leave out this line completely? (no idea if an "organization" is important for the citation if there is an author already)

Reply:
yes, leave it out.